Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Requirements for container registry
+type: Decision Record
+status: Draft
+track: KaaS
+---
+
+# Introduction  
+
+A container registry is a repository or collection of repositories used to store and 
+access container images. Container registries are used by the teams to store the 
+results of build processes and to maintain freely available images. They're acting as 
+the intermediary for sharing container images between systems. They could be connected 
+directly to the container orchestration platforms like Kubernetes.
+
+The container registries could be publicly accessible e.g. Docker Hub or could be
+self-hosted or hosted by the cloud service providers (CSP). These container registries
```

Comment:
I would differentiate between `CSP` (singular) and `CSPs` (plural) here and remove `the` from `the cloud service providers`.